docs/user-guide/reduction-workflow-guidelines.md

```diff
@@ -122,6 +122,31 @@ Users should not have to worry about the concrete type of parameters.
   E.g., `sc.scalar(2, unit='m')` and `sc.scalar(2.0, unit='m')` should be usable interchangeably.
   This can also apply to arrays, for instance, `sc.linspace` and `sc.arange` should be interchangeable but the latter may result in integers while the former typically produces floats.
 
+### C.5: Use a conventional pattern minimizing programming jargon, favor fewer and concepts
+
+**Reason**
+- Terms such as provider or pipeline mean nothing to users, using their language instead reduces cognitive load.
```

Comment:
```suggestion
- Terms such as provider or pipeline mean nothing to users, using their natural language and instead reduces cognitive load.
```

Reply:
Their natural language is English, I was referring to how they are used to referring to things in their work/research.

Reply:
That does not parse grammatically.

Reply:
True. Let's be more explicit:
```
Using terms such as provider or pipeline increases cognitive load for scientific users.
The reason is that those terms are unfamiliar to users based on the scientific domain language they are used to.
```

Reply:
```suggestion
- Using terms such as provider or pipeline increases cognitive load for scientific users.
  The reason is that those terms are unfamiliar to users based on the scientific domain language they are used to.
```